ga4gh/backend.py

```diff
@@ -413,6 +428,24 @@ def getVariantSet(self, id_):
         dataset = self._getDatasetFromCompoundId(id_)
         return self.runGetRequest(dataset.getVariantSetIdMap(), id_)
 
+    def getVariant(self, id_):
+        """
+        Returns a variant with the given id
+        """
+        try:
+            splits = id_.split(':')
+        except AttributeError:
+            raise exceptions.BadIdentifierException(id_)
+        try:
+            variantSetId = '{}:{}'.format(splits[0], splits[1])
+        except KeyError:
+            raise exceptions.BadIdentifierException(id_)
+        dataset = self._getDatasetFromCompoundId(id_)
+        variantSet = _safeMapQuery(
+            dataset.getVariantSetIdMap(), variantSetId)
+        variantId = ':'.join(splits[2:])
+        raise Exception(variantSet + variantId)  # TODO
```

Comment:
@jeromekelleher what does the variant's "id" mean in our realm?  In `datamodel/variants.py` we return `variantSetId:record.contig:record.pos` as the id.  However, the VCF format has an "ID" for each record (that may or may not be there).  Which are we using, or is there some other concept of id?  Is this id guaranteed to be unique?  

And as far as accessing the VCF file goes, is the algorithm just going to be iterate and stop when the first matching variant is seen and return it?